text/0000-temp-components-lib.md

```diff
@@ -0,0 +1,610 @@
+a handfull of new people interested in helping contribute to a component library
+which would also speed things up and reduce cost.
+
+Can this library be implemented in our users space? Yes, since most of our teams
+choose to use React, it should only help. However, it should be noted that this
+creates favoritism to React which is typically a negative point since engineering
+should never use tooling from a bias but instead as a result of what is the
+"best tool for the job". It is literally impossible that React will always be the
+best tool for Pluralsight (and it may already be so, we are just choosing not to
+see it?). By nature, one day this specific library will be obsolete.
+
+What would be the impact on teaching people about Pando React? Minimal. The biggest
+hurdle will probably be teaching people about more advanced/preferred React patterns
+based on performance and code scalability. This is only due to the larger amount
+of product code, 3rd party components, and libraries that exist in the world that
+do not use good React practices/anti-patterns (i.e. all in one components, etc.).
+However, I don't think this will really be that big of a deal?
+
+What about integration of this library with other existing features? Everything
+in the [Pando Ecosystem](https://github.com/pluralsight/pando/tree/main/packages/headless-styles#what-role-does-this-package-play-in-pando) should naturally be able
+to co-exists without requiring any additional work from the developer experience.
+This also fulfills our [Commitment to Stability](https://design.pluralsight.com/docs/next/learn/about-pando/versioning#commitment-to-stability).
+
+What is the cost of migrating to Pando React from existing solutions (including Classic)?
+By nature, Pando will be a breaking change since it is fundamentally different down
+to the Web UI Kit (design options). However, it should allow the users to migrate
+easier since it would mostly be updating prop names and values on already existing components.
+Likewise, it should allow the users depenedency list to dramatically decrease since
+they will only need a single package `@pluralsight/react` vs. a package for every single
+component used and it's required dependecies (i.e. Classic, MUI, Chakra, etc.).
+
+Ultimately, the pro's seem to outweigh the cons as long as users understand this
+library is not meant to be the end-game to Pando (Headless-styles is). Headless-styles
+will allow us to continue to scale throughout the trends of technology shifts by
+nature of it being a vanilla Javascript library.
+
+## Alternatives
+
+We've attempted to use more advanced/current/modern designs via Headless-styles,
+however, we have learned that ultimately, that design is still too advanced for
+the majority of Pando users. This is due to it relying on the user being comfortable
+and confident in front-end technologies [which isn't always the case we have found](#what-use-cases-does-it-support).
+
```

Comment:
To be completely honest, this part feels a little condescending. I know that I personally am very comfortable and confident in front-end technologies, and I may have been one of the loudest voices in support of a component library 🙃 Like I mentioned elsewhere, I think the more common situation is that the amount of effort and time required to introduce this new paradigm into existing code is just hard to justify from a time/cost perspective, rather than it being to advanced to understand or implement.

Reply:
@josh-degraw , I appreciate that feedback. Let me update it to use the language you are saying so it doesn't come across that way.